flybywiresim/aircraft
# fix(nd): font size and ts errors

Fixes #8087

## Summary of Changes

While finishing the NDv2, the font size of the approach message was changed to be more accurate. But other elements in the upper region on the ND were untouched. This adapts the font sizes of the remaining elements to match the approach message. 

Additionally, overlapping Range/Mode changes should be fixed not appear when MAP NOT AVAIL flag is being shown

There was also an error introduced in the PFD which is fixed now.

## Screenshots (if necessary)


![image](https://github.com/flybywiresim/a32nx/assets/19493808/43ca9806-21d4-473d-b14c-0366e62471f8)

## References






## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Before aligning, switch ND modes and range and make sure they do not appear.
2. After aligning, do the same and ensure they do not overlap (range change message should override mode change message)
3. Make sure everything in the upper region of the ND is available and readable (GS/TAS/TO WPT Indication (distance/ETA...)
4. Turn off the FDs, ensure you are in selected heading and verify the heading indicator on the PFD shows up (given the selected heading is in range)


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
